README.rst

```diff
@@ -29,26 +29,24 @@ Use at the command line:
 
 1. In your current working directory type ``source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate``
 
-2. To check if a PV or device name is valid pipe in the name to the tool. For example, ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py``
+2. To check if a PV or device name is valid pipe in the name to the tool. For example, ``$ echo 'MR2K4:KBO:PIP:01:PUMPSIZE' | ./lcls_naming_tool.py``
```

Comment:
Nitpick: all the readme examples use the stdin piping syntax. I think we should advertise the cli syntax from using argparse first or in addition to the piping syntax

Reply:
I just realized today that argparse wasn't set up to read files, so spent some time learning how to do this. It now accepts files with the cli syntax and a `-f` flag (example: `python lcls_naming_tool.py -f pvlist.txt`).